tests/system/system_tests.cfg

```diff
@@ -1,8 +1,214 @@
 [System-check-domainfile-and-nans]
 driver = image
 test_description = check that coordinates and mask in domain and output files match
-global_parameter_file = global_param.image.STEHE.feb.txt
-domain_file = domain.stehekin.20151028.nc
+global_parameter_file = global.image.STEHE.txt
+domain_file = image/Stehekin/parameters/domain.stehekin.20151028.nc
 mpi_proc = 4
 expected_retval = 0
 check = output_file_nans
+
+[System-check_classic_nans]
+driver = classic
+test_description = Short simulation to ensure system tests configuration works - classic driver
+global_parameter_file = global.classic.STEHE.txt
+expected_retval = 0
+check = output_file_nans
+
+[System-restart_classic_noFullEnergy_noFrozenSoil]
+# test_description = Exact restart (falseFULL_ENERGY flaseFROZEN_SOIL) - classic driver
+driver = classic
+global_parameter_file = global.classic.STEHE.restart.txt
+expected_retval = 0
+check = exact_restart
+[[restart]]
+# If test_type == exact_restart
+# Full period - start and end date
+# Full period run will start with no initial state file, and the last time step state will be saved
+start_date = 1949,1,1
+end_date = 1949,1,10
+# Dates to split the run
+# E.g., if the full running period is 1949-01-01 to 1949-01-10, and split at 1949-01-05, then the first run would be 1949-01-01 to 1949-01-05, the second run would be 1949-01-06 to 1949-01-10
+# There can be multiple splitting dates
+[[[split_dates]]]
+split_date_1 = 1949,1,5
```

Comment:
I think we can simplify this format a bit.  

```
[System-restart_classic_noFullEnergy_noFrozenSoil]
test_description = Exact restart (falseFULL_ENERGY flaseFROZEN_SOIL) - classic driver
driver = classic
global_parameter_file = global.classic.STEHE.restart.txt
expected_retval = 0
check = exact_restart
[[restart]]
start_date = 1949-01-01
end_date = 1949-01-10
split_dates = 1949-01-04, 1949-01-08
[[options]]
FULL_ENERGY=FALSE
FROZEN_SOIL=FALSE
```

In fact, we don't have to have the `[[restart]]` section at all but since it will only be used when `check = exact_restart`, I think its ok.